src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10364,30 +10311,23 @@ private static bool IsAccessibilityModifier(SyntaxKind kind)
                     case SyntaxKind.VolatileKeyword:
                         continue; // already reported earlier, no need to report again
                     case SyntaxKind.StaticKeyword:
-                        modifier = CheckFeatureAvailability(modifier, MessageID.IDS_FeatureStaticLocalFunctions);
-                        if ((object)modifier == modifiers[i])
-                        {
-                            continue;
-                        }
-                        break;
```

Comment:
It looks like there's a logic change here. Is this intentional?

Reply:
the old logic said "if i didn't update the modifier, continue.  if i did, break".  As we no longer update hte modifier ever here, we always continue.